src/renderer/components/general-settings/general-settings.vue

```diff
@@ -13,6 +13,7 @@
         <ft-toggle-switch
           :label="$t('Settings.General Settings.Fallback to Non-Preferred Backend on Failure')"
           :default-value="backendFallback"
+          :disabled="backendPreference === 'piped'"
```

Comment:
**suggestion:** Add an additional sentence explaining why it's disabled for Piped, either to this tooltip or the one for Preferred API Backend. More broadly, would like to better convey to people what the limitations of the Piped API primary backend are.

Reply:
done